src/assets/scss/elements/_login_register.scss

```diff
@@ -0,0 +1,264 @@
+/* Login Register page css */
+.container {
```

Comment:
rename all classes or delete if you don't use

Reply:
done